submissions/lawalLolade/html-forms/contactFormPage.html

```diff
@@ -0,0 +1,79 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flawa | Contant Us</title>
+</head>
+<body>
+    <!-- header Begins -->
+<header>
+    <div>
+        <h1>FLAWA</h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="../html/productPage.html">Product</a>
+                </li>
+                <li><a href="contactFormPage.html">Contact Us</a></li>
+                <li><a href="#">Garden</a></li>
+                <li><a href="#">Vases</a></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+    <!-- Header ends -->
+
+
+    <!-- Contact Us Form -->
+<section>
+    <div>
+        <form action="">
+            <fieldset>
+                <legend><h2>Contact Us</h2></legend>
+                <div>
+                    <label for="firstname"><strong>First Name:</strong> </label>&nbsp;&nbsp;&nbsp;
+                    <input type="text" name="firstname" id="firstname">
+                </div><br>
+                <div>
+                    <label for="lastname"><strong>Last Name:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;
+                    <input type="text" name="lastname" id="lastname">
+                </div><br>
+                <div>
+                    <label for="email"><strong>Email:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                    <input type="email" name="email" id="email">                    
+                </div><br>
+                <div>
+                    <label><strong>Subject:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Kindly remove the extra characters after the closing tag. The section is also not properly formatted.